backend/api/oauthEnabledClient.js

```diff
@@ -26,86 +31,102 @@ const errorLogger = error => {
  * @returns {{get: (function(*=): *), post: (function(*=, *=): *)}}
  */
 const factory = ({ baseUrl, timeout }) => {
+  // strip off any excess / from the required url
+  const remoteUrl = baseUrl.endsWith('/') ? baseUrl.substring(0, baseUrl.length - 1) : baseUrl
```

Comment:
extra logic to ensure that env variables don't end with a `/` so that paths can instead